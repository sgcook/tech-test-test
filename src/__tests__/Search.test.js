import React from "react";
import { render } from "@testing-library/react";
import Search from "../components/Search";
import { func } from "prop-types";

describe("Search", () => {
  const validProps = {
    setSearchResults: () => {},
  }
  it("renders correctly", () => {
    const { asFragment } = render(
      <Search setSearchResults={validProps.setSearchResults} />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});