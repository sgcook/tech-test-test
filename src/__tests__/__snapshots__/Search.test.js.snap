// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders correctly 1`] = `
<DocumentFragment>
  <form
    class="search-form"
  >
    <input
      class="search-input"
      placeholder="Search here!"
      type="text"
    />
    <button
      class="search-btn"
      type="submit"
    >
      Go!
    </button>
  </form>
</DocumentFragment>
`;
